Blockly - single stop exec

n8xm opened the pull request:
We can now stop execution on a particular Blockly node,as opposed to just one of them.

Commit:
we can now stop execution on a particular Blockly node,as opposed to
just one of them.

BrettASwift commented on support/proxy/vwf.example.com/blockly/manager.vwf.yaml:
```diff
@@ -27,6 +27,6 @@ properties:
 methods:
   startAllExecution:
   stopAllExecution:
+  stopExecutionForNode:
```
It might be better to put a `stopExecution` method on the `blockly/controller.vwf` component instead. It makes more sense. Then we won't need to pass the ID. The functionality should be almost identical.

Commit:
Now, per Brett's suggestion, we changed stopExecutingForNode in
manager.vwf.yaml to stopExecuting in controller.vwf.yaml. This way we
don't have to pass the node ID as an argument.

BrettASwift commented:
Looks good. :+1:
